ScaleFast.roboFontExt/lib/scaleFast.py

```diff
@@ -594,7 +596,7 @@ def _setTargetHeight(self, sender):
                 height_targetValue = height_referenceValue * float(targetHeight)
                 self.controls.verticalScaleBox.targetHeightInput.set(int(round(height_targetValue)))
 
-            elif isinstance(sender, EditText) and len(targetHeight):
+            elif isinstance(sender, EditText) and len(str(targetHeight)):
```

Comment:
you can set `NumberEditText` to `allowEmpty=False`

I guess this checks if there is a value provided instead of an empty string